app/views/works/_required_form.html.erb

```diff
@@ -8,7 +8,7 @@
   <% if @wizard_mode || !doi_mutable? %>
     <input id="doi_read_only" name="doi_read_only" value="<%= @work.resource.doi %>" readonly/>
   <% else %>
-    <p id="doi">DOI: <%= @work.resource.doi %></p>
+    <p id="doi_text">DOI: <%= @work.resource.doi %></p>
```

Comment:
We should not have two HTML elements with the same id, so I made the paragraph have a different id so that the JavaScript applies to the input box, not to the paragraph.